Sources/SwiftParser/Attributes.swift

```diff
@@ -176,7 +176,14 @@ extension Parser {
     argumentMode: AttributeArgumentMode,
     parseArguments: (inout Parser) -> RawAttributeSyntax.Arguments
   ) -> RawAttributeListSyntax.Element {
-    let (unexpectedBeforeAtSign, atSign) = self.expect(.atSign)
+    var (unexpectedBeforeAtSign, atSign) = self.expect(.atSign)
+    if atSign.trailingTriviaByteLength > 0 || self.currentToken.leadingTriviaByteLength > 0 {
```

Comment:
Does it make sene to add `unexpectedBeforeAtSign  == nil` here? For `attrName<unexpected> ( ...`. Diagnosing trivia between `attrName` and `(` doesn't make much sense.